### PR TITLE
Update to Flexible Voting v1.1.0

### PR DESCRIPTION
**Pull request by apbendi**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/foundry-rs/forge-std
 [submodule "lib/flexible-voting"]
 	path = lib/flexible-voting
-	url = https://github.com/ScopeLift/flexible-voting
+	url = https://github.com/scopelift/flexible-voting
```
Comment (apbendi) [inline .gitmodules:6]: Not sure why git changed the casing here 🤷‍♂️

Comment (apbendi) [reply .gitmodules:6]: `git submodule status` now returns

```
 e5de2efd1368387b840931f19f3c184c85842761 lib/flexible-voting (v1.1.0)
 fc560fa34fa12a335a50c35d92e55a6628ca467c lib/forge-std (v1.5.3)
 1b3adf677e7e383cc684b5d5bd441da86bf4bf1c lib/solmate (v6-186-g1b3adf6)
```

File: test/GitcoinGovernor.t.sol [hunk 2]
```diff
@@ -20,6 +20,7 @@ abstract contract GitcoinGovernorTestHelper is Test, DeployInput {
   address constant TIMELOCK = 0x57a8865cfB1eCEf7253c27da6B4BC3dAEE5Be518;
   address constant PROPOSER = 0xc2E2B715d9e302947Ec7e312fd2384b5a1296099; // kbw.eth
   address constant DEPLOYED_BRAVO_GOVERNOR = 0x1a84384e1f1b12D53E60C8C528178dC87767b488;
+  uint256 constant MAX_REASONABLE_TIME_PERIOD = 302_400; // 6 weeks assuming a 12 second block time
```
Comment (apbendi) [inline test/GitcoinGovernor.t.sol:23]: We added this sanity check to PoolTogether repo and it seemed worthwhile to add them here as well

File: foundry.toml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 [profile.default]
   optimizer = true
   optimizer_runs = 10_000_000
+  remappings = ["@openzeppelin=lib/flexible-voting/lib/openzeppelin-contracts/"]
```
Comment (apbendi) [inline foundry.toml:4]: repo wouldn't build with the latest version of foundry until I added this

File: .gitmodules (hunk 1 above)
Comment (alexkeating) [reply .gitmodules:6]: I have seen this after running a forge install with a path that has a different casing

File: foundry.toml (hunk 3 above)
Comment (alexkeating) [reply foundry.toml:4]: Yes, I saw this with a bunch of my repos as well

File: .gitmodules (hunk 1 above)
Comment (alexkeating) [reply .gitmodules:6]: I ran forge install and am getting 
Screenshot from 2023-07-27 19-51-30

Comment (alexkeating) [reply .gitmodules:6]: The hash is right but tag is wrong for some reason maybe I have a different foundry version

**Review by alexkeating — APPROVED** [review]
LGTM

File: foundry.toml (hunk 3 above)
Comment (mds1) [reply foundry.toml:4]: for reference I think this is related to https://github.com/foundry-rs/foundry/issues/5475

**Review by mds1 — APPROVED** [review]

**Pull request merged**